plugins/rpm_conf.py

```diff
@@ -43,5 +68,23 @@ def resolved(self):
                 self.packages.append(pkg.name)
 
     def transaction(self):
-        rconf = rpmconf.RpmConf(packages=self.packages)
-        rconf.run()
+        if not self._interactive:
+            logger.debug(_("rpmconf plugin will not run "
+                           "in non-interactive mode"))
+            return
+
+        rconf = rpmconf.RpmConf(
+            packages=self.packages,
+            frontend=self.frontend,
+            diff=self.diff)
+        try:
+            rconf.run()
+        except SystemExit as e:
+            if e.code == 2:
```

Comment:
by the way [rpmconf module got them as numbers] (https://github.com/xsuchy/rpmconf/blob/f0543ced24227f5bb2ebe591278ae27c5a8d34ce/rpmconf/rpmconf.py#L275-L281)